.remarkrc.js

```diff
@@ -40,15 +51,37 @@ exports.plugins = [
     { no: 'cripple' },
     { no: 'sanity(-|\s)check' },
     { no: 'grandfathered' },
+    { no: '(grandfather)(-|\s)?(clause|ed|ing)' },
+    { no: '(mom|girlfriend|grandma|grandmother)(-|\s)?(test)' },
+    { no: '(build)(-|\s)?(cop|sheriff)' },
+    { no: 'ADD' },
+    { no: 'open(-|\s)?the(-|\s)?kimono' },
     { no: 's?he' },
+    { no: 'final(-|\s)?solution'},
+    { no: 'WTF(-|\s)?error'},
+    { no: 'ninja' },
+    { no: 'guru' },
     { no: '^h(is|er(s?))$' },
     { no: 'man-in-the-middle' },
     { no: 'mitm' },
     { no: 'redline' },
     { no: '(black|white|gray|grey)(-|\s)?(hat|list|glove|label)' },
     { no: 'rtfm' },
     { no: 'wtf' }],
-    { no: 'post mortem' }
+    { no: 'post mortem' },
+    { no: 'retarded' },
+    { no: 'demented' },
+    { no: 'developmentally(-|\s)?disabled' },
+    { no: 'idiot' },
+    { no: 'mongoloid' },
+    { no: 'permenant(-|\s)?child' },
+    { no: 'short(-|\s)?bus' },
+    { no: 'slow(-|\s)?learner' },
+    { no: 'daft' },
+    { no: 'imbecil' },
+    { no: 'stupid' },
+    { no: 'moronic' },
+    { no: '(-|\s)?tard' }
```

Comment:
```suggestion
    { no: '(-|\s)?tard(ed)?' }
```
Adding this at the end should cover everything that ends in `tard` or `tarded`, and let us get rid of stand-alone `retarded`